dist/common/scripts/scylla_setup

```diff
@@ -550,6 +560,14 @@ if __name__ == '__main__':
 
                 run_setup_script('swap', 'scylla_swap_setup' + options)
 
+        if os.path.isdir('/etc/rsyslog.d'):
+            rsyslog_setup = interactive_ask_service('Do you want to configure rsyslog to send log to a remote repository?', 'Answer yes to setup rsyslog to a remote server, Answer no to do nothing.', rsyslog_setup)
+            args.no_rsyslog_setup = not rsyslog_setup
+            if rsyslog_setup:
+                if interactive:
+                    rsyslog_server = interactive_ask_rsyslog_server()
+                run_setup_script('rsyslog', 'scylla_rsyslog_setup --remote-server ' + rsyslog_server)
```

Comment:
This will crash if someone uses the non-interactive setup and forgets to use --remote-server (like using an existing script). We should skip if --remote-server was not specified.

Reply:
I found it confusing but I followed what happens in swap configuration.

Originally, I only looked at remove-server and if it was empty, skip, but following @syuu1228 comment changed to the current behavior

Reply:
Looks at both, or we make random ansible scripts breaks.